### PR TITLE
fix: actions are more robust

### PR DESCRIPTION
**Pull request by drazik**
There were some problems on actions on mobile (wrong button type in the modal mostly).

I also fixed some little things after Joel's design review (transaction row padding, category icon size).

### CODE REVIEW

File: src/ducks/transactions/TransactionActions.styl [hunk 1]
```diff
@@ -20,3 +20,13 @@
...
+
+.TransactionActionButton
+    +medium-screen()
+        &:before
+            content ''
+            position absolute
+            top 0
+            bottom 0
+            width 1rem
+            right 100%
```
Comment (ptbrowne) [inline src/ducks/transactions/TransactionActions.styl]: can you add comment saying this is to enlarge the click area (if I've correctly understood)

File: src/ducks/transactions/Transactions.styl [hunk 2]
```diff
@@ -26,16 +26,16 @@
     .bnk-transaction-mobile
         & > td
             width 100%
+            padding-top .75rem
+            padding-bottom .75rem
```
Comment (ptbrowne) [inline src/ducks/transactions/Transactions.styl]: why is this necessary ?

Comment (drazik) [reply src/ducks/transactions/Transactions.styl]: Because the base vertical padding on tables is `1rem`, but here we want the padding to be a little bit smaller.
